server/graphql/v1/types.js

```diff
@@ -241,6 +241,12 @@ export const UserType = new GraphQLObjectType({
           return user.data && user.data.features && user.data.features.ALL === false;
         },
       },
+      changelogViewDate: {
+        type: DateString,
```

Comment:
Haven't used these in quite some time (GQLV1) but I think `IsoDateString` better respects standards

```suggestion
        type: IsoDateString,
```